<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcwidget</class>
 <widget class="QWidget" name="calcwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calcwidget</string>
  </property>
  <widget class="QGroupBox" name="operationBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>120</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Операция</string>
   </property>
   <widget class="QRadioButton" name="plusR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minusR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="multiplicationR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="divisionR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sqrtR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>^1/2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sinR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cosR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="doButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QLabel" name="num1Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Число 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="num2Label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Число 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num1Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="num2Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ошибка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
